src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -948,18 +1038,24 @@ private bool IsFailOverable(AggregateException ex)
 
         private bool IsFailOverable(RequestFailedException rfe)
```

Comment:
For net framework 4.8 only, tests failed because the structure of the `RequestFailedException` returned was different, with the inner exceptions not being wrapped in an `HttpRequestException`. I've adjusted the `IsFailoverable` method to account for that. Wanted to call it out here. 

Actual:   typeof(System.AggregateException): Retry failed after 3 tries. Retry settings can be adjusted in ClientOptions.Retry or by configuring a custom retry policy in ClientOptions.RetryPolicy.
---- System.AggregateException : Retry failed after 3 tries. Retry settings can be adjusted in ClientOptions.Retry or by configuring a custom retry policy in ClientOptions.RetryPolicy.
-------- Azure.RequestFailedException : The remote name could not be resolved: 'azure.azconfig.io'
------------ System.Net.WebException : The remote name could not be resolved: 'azure.azconfig.io'
-------- Azure.RequestFailedException : The remote name could not be resolved: 'azure.azconfig.io'
------------ System.Net.WebException : The remote name could not be resolved: 'azure.azconfig.io'
-------- Azure.RequestFailedException : The remote name could not be resolved: 'azure.azconfig.io'
------------ System.Net.WebException : The remote name could not be resolved: 'azure.azconfig.io'